app_connections/models/review.model.js

```diff
@@ -1,21 +1,45 @@
 const db = require("../../db/connection");
 
-exports.fetchAllReviews = (category) => {
+exports.fetchAllReviews = (
+  category,
+  sort_by = "created_at",
+  order = "DESC"
+) => {
   const queryValue = [];
+  const tester = order.toUpperCase();
+  const validOrderList = ["ASC", "DESC"];
+  const validSortList = [
+    "review_id",
+    "title",
+    "designer",
+    "owner",
+    "review_img_url",
+    "review_body",
+    "category",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+  if (!validSortList.includes(sort_by) || !validOrderList.includes(tester)) {
+    return Promise.reject({ status: 400, msg: "invalid value given" });
+  }
```

Comment:
You can consider separating those two errors to have a more specific error message, but is all up to you!